documentation/binding-data/tutorial-flow-components-binder-beans.asciidoc

```diff
@@ -228,4 +272,6 @@ binder.bindInstanceFields(this);
 
 ----
 
-The `bindInstanceFields` does not support validations and if you want to add validation you need to use the `BeanValidationBinder` instead of the `bindInstanceFields`.
+=== Validating Fields
+
+The `bindInstanceFields` method does not support validation. If you want to add validation, use `BeanValidationBinder` instead of the `bindInstanceFields` method.
```

Comment:
AFAIK this is (and was) not true. bindInstanceFields don't support programmatic validation, but it sure does support JSR-303 validation. When you are using that, you need to use BeanValidationBinder anyways and then it is just fine to use bindInstanceFields method. 

The whole section also should be improved (by to core team first as they know how it works). I think this should underline that you can use this method together with manual binding, if you need custom programatic validator for certain properties, but the rest are fine. I'll try to rephrase this, but @pleku must check that this becomes valid.

```suggestion
The `bindInstanceFields` method does not support adding programmatic validation bindings. If you are using JSR-303 validators, they work fine when using `BeanValidationBinder`. Also, you can bind properties that need validators manually and then bind all remaining fields using bindInstanceFields method.
```

@pleku Do we completely miss documentation for the "forMemberField" style configuration? I think I have used that and it looks like it supports also configuring validators, making this still plain FUD.